devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -30,6 +30,7 @@
       - python-cornice
       - python-cornice-sphinx
       - python-devel
+      - python2-diff-cover
```

Comment:
I think I lied to you - apparently this package doesn't provide ```/usr/bin/diff-cover``` - only ```python3-diff-cover``` seems to provide that. The tool does seem to work as you documented with the python3 version though.